ruma-events/src/lib.rs

```diff
@@ -255,6 +266,10 @@ pub struct EventDeHelper {
     /// If no `event_id` or `state_key` are found but a `room_id` is present
     /// the event will be deserialized as a ephemeral event.
     pub room_id: Option<IgnoredAny>,
+
+    /// If this `UnsignedData` contains a redacted_because key the event is
+    /// immediately deserialized as a redacted event.
+    pub unsigned: Option<UnsignedData>,
```

Comment:
We should use a separate type here, sth like `struct UnsignedDeHelper { redacted_because: Option<IgnoredAny> }`.